src/hearings/components/multi-level-selector/multi-level-selector.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from '@angular/core';
+import { FormArray, FormBuilder, FormGroup } from '@angular/forms';
+import { ControlTypeEnum } from 'src/hearings/models/hearings.enum';
+
+@Component({
+  selector: 'exui-multi-level-selector',
+  templateUrl: './multi-level-selector.component.html',
+  styles: ['.multi-select-container { margin-top: 10px; }']
+})
+export class MultiLevelSelectorComponent {
+  @Input() public multiSelect: FormArray;
```

Comment:
if this is also supporting radio, would this still call multiSelect?

Reply:
I should off been multLevelSelect. I will change to that instead.